bundles/org.jkiss.utils/src/org/jkiss/utils/ByteNumberFormat.java

```diff
@@ -163,4 +163,9 @@ private enum Unit {
             this.fullName = fullName;
         }
     }
-}
\ No newline at end of file
+
+    public enum UnitNames {
```

Comment:
Perhaps `UnitNaming` would be more convenient?

Reply:
I don´t see the point tbqh

Reply:
Or `UnitStyle` to be consistent with the `java.time.format` API naming scheme because I like it.

Reply:
I think I'll change the name to match the language used in ISO 80000-13.